pkg/backoff/backoff.go

```diff
@@ -16,8 +16,8 @@ import (
 )
 
 const (
-	// resetAfter sets the total seconds a backoff will last.
-	resetAfter = 3600
+	// maxBackoff sets the maximum time we will rate limit before forcing to resend.
+	maxBackoff = time.Hour
```

Comment:
let's not use `time.Hour` here to be more consistent acrros the code. What about keeping 3600 seconds?
```suggestion
	maxBackoffSecs = 3600
```

Reply:
`time.Hour` conveys meaning, but `3600` doesn't. Let's start changing everywhere else in the codebase to use Durations instead of integers?

Reply:
`time.Hour` represents a Duration and it adds more complexity to a simple variable to control the time. Having an integer is much simpler and consumes less memory and it also don't keep the whole struct in the heap.

Reply:
Any memory side effect is so minimal it won't matter compared to code readability. I think using a duration makes it easier to read and clear vs integer.